src/Interface.mo

```diff
@@ -47,7 +47,7 @@ module {
         // Returns the account that is linked to the given token.
         bearer  : query (token : Ext.TokenIdentifier)            -> async Ext.NonFungible.BearerResponse;
         // Mints a new NFT and assignes its owner to the given User.
-        mintNFT : shared (request : Ext.NonFungible.MintRequest) -> async ();
+        mintNFT : shared (request : Ext.NonFungible.MintRequest) -> async Ext.NonFungible.MintResponse;
```

Comment:
I think we should just return `TokenIndex` instead of a `Result` type, like in the [example implementation](https://github.com/Toniq-Labs/extendable-token/blob/1f7ef3e2cff3f95509f607ba022326fe280f7039/examples/erc721.mo#L74).

Reply:
I would personally keep it similar to the rest of the responses.
The minting could result in various errors: Not Allowed, Meta too large, Invalid To address.
Its your call tho. Whatever you say.